CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+### Features
+
+- Support beforeSendTransaction ([#1238](https://github.com/getsentry/sentry-dart/pull/1238))
+  - Add `fingerprint` parameter to `SentryTransaction` constructor
+  - Add `modules` parameter to `SentryTransaction` constructor
```

Comment:
Can we double-check if sentry-java or sentry-cocoa also sends or does not send those 2 values for transactions? I don't remember by heart, @romtsn or @philipphofmann can help.

Reply:
I guess those are not part of transaction. In `sentry-java`, the event and transaction are both subclasses of a base event. They do not share [`fingerprint`](https://github.com/getsentry/sentry-java/blob/f6a135d1ef81f9cd70d72608e8beb8c2103f69bd/sentry/src/main/java/io/sentry/SentryEvent.java#L69) and `modules`. Those are only part of the event. I'll remove those again and also remove them from the test expectation.